client/app/requests/requests-state.service.js

```diff
@@ -1,8 +1,19 @@
 /** @ngInject */
-export function RequestsStateFactory(ListConfiguration) {
-  var service = {};
+export function RequestsStateFactory(ListConfiguration, RBAC) {
+  var service = {
+    getPermissions: getPermissions,
+  };
 
   ListConfiguration.setupListFunctions(service, { id: 'requested', title: __('Request Date'), sortType: 'numeric' });
 
+  function getPermissions() {
```

Comment:
serious question... thinking that maybe not... because an admin. will have these checked... but.... not positive...